InternalDocs/asyncio.md

```diff
@@ -2,10 +2,10 @@ asyncio
 =======
 
 
-This document describes the working and implementation details of C
-implementation of the
+This document describes the working and implementation details of the
 [`asyncio`](https://docs.python.org/3/library/asyncio.html) module.
 
+# Task management
```

Comment:
IIUC this section only discusses the C version -- can you call that out somehow? Previously that was implied by the opening sentence above but that has changed now.

Reply:
Added a sentence before both section to clarify that.

Reply:
done